src/api/public/apidocs-new/paths/search_owner.yaml

```
@@ -0,0 +1,100 @@
+get:
+  summary: Search for the default responsible person or group (the owner).
+  description: |
+    Search for the default responsible person or group (the owner) by a user, group
+    or binary name.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: binary
+      schema:
+        type: string
+      required: true
+      description: |
+        The name of a build binary. Either the `binary`, `user` or `group`
+        parameter must be provided.
```

```suggestion
      description: |
        The name of a build binary. Either the `binary`, `user` or `group`
        parameter must be provided.
```
AFAIK, the OpenAPI specification doesn't support one of a set of parameters to be required.

In our case, requiring the three parameters doesn't solve the issue. It would force to provide values for each of the three parameters.

I suggest to remove the `required: true` from the `binary`, `user` and `group` parameters, keeping the description in all of them.